store/src/java/com/zimbra/cs/event/logger/EventLoggerUtils.java

```diff
@@ -0,0 +1,25 @@
+package com.zimbra.cs.event.logger;
+
+import com.zimbra.cs.event.Event;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class EventLoggerUtils {
+    public static List<Event> getEventForEachRecipient(Event event) {
+        List<Event> events = new ArrayList<>();
+        String receiver = (String) event.getContextField(Event.EventContextField.RECEIVER);
+        String[] recipients = receiver.split(Event.MULTI_VALUE_SEPARATOR);
+        if (recipients.length > 1) {
```

Comment:
Not >= 1?

Reply:
The else block handles the case when there is only one recipient.

Reply:
Ah.  Sorry, daftness.